scene/surface-placement/src/main/java/com/esri/samples/surface_placement/SurfacePlacementController.java

```diff
@@ -63,73 +63,41 @@ public void initialize() {
       relativeToSceneOverlay.getSceneProperties().setSurfacePlacement(LayerSceneProperties.SurfacePlacement.RELATIVE_TO_SCENE);
-
-      // create points for graphic locations
-      Point surfaceRelatedPoint = new Point(-4.4609257, 48.3903965, 70, SpatialReferences.getWgs84());
-      Point sceneRelatedPoint = new Point(-4.4610562, 48.3902727, 70, SpatialReferences.getWgs84());
-
-      // create a red triangle symbol
-      SimpleMarkerSymbol triangleSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.TRIANGLE, 0xFFFF0000, 12);
-
-      // create a text symbol for each surface placement type
-      TextSymbol drapedBillboardedText =
-          new TextSymbol(14, "DRAPED BILLBOARDED", 0xFF0000FF, TextSymbol.HorizontalAlignment.LEFT, TextSymbol.VerticalAlignment.MIDDLE);
-      drapedBillboardedText.setOffsetX(20);
-
-      TextSymbol drapedFlatText =
-          new TextSymbol(14, "DRAPED FLAT", 0xFF0000FF, TextSymbol.HorizontalAlignment.LEFT, TextSymbol.VerticalAlignment.MIDDLE);
-      drapedFlatText.setOffsetX(20);
-
-      TextSymbol relativeText =
-          new TextSymbol(14, "RELATIVE", 0xFF0000FF, TextSymbol.HorizontalAlignment.LEFT, TextSymbol.VerticalAlignment.MIDDLE);
-      relativeText.setOffsetX(20);
-
-      TextSymbol absoluteText =
-          new TextSymbol(14, "ABSOLUTE", 0xFF0000FF, TextSymbol.HorizontalAlignment.LEFT, TextSymbol.VerticalAlignment.MIDDLE);
-      absoluteText.setOffsetX(20);
-
-      TextSymbol relativeToSceneText =
-        new TextSymbol(14, "RELATIVE TO SCENE", 0xFF0000FF, TextSymbol.HorizontalAlignment.RIGHT, TextSymbol.VerticalAlignment.MIDDLE);
-      relativeToSceneText.setOffsetX(-20);
-
-      // add the point graphic and text graphic to the corresponding graphics overlay
-      drapedBillboardedOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, triangleSymbol));
-      drapedBillboardedOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, drapedBillboardedText));
-
-      drapedFlatOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, triangleSymbol));
-      drapedFlatOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, drapedFlatText));
-
-      relativeOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, triangleSymbol));
-      relativeOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, relativeText));
-
-      absoluteOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, triangleSymbol));
-      absoluteOverlay.getGraphics().add(new Graphic(surfaceRelatedPoint, absoluteText));
-
-      relativeToSceneOverlay.getGraphics().add(new Graphic(sceneRelatedPoint, triangleSymbol));
-      relativeToSceneOverlay.getGraphics().add(new Graphic(sceneRelatedPoint, relativeToSceneText));
+      addGraphicsToGraphicsOverlay(relativeToSceneOverlay, relativeToSceneText);
 
       // add graphics overlays to the scene view
       sceneView.getGraphicsOverlays().addAll(Arrays.asList(drapedBillboardedOverlay, relativeOverlay, absoluteOverlay, relativeToSceneOverlay));
```

Comment:
I've just noticed that this line goes over our max line length guidance (120 chars). We should split it up to avoid users having to use the horizontal scroll bar on Github (another easy to implement improve readability thing)! 

```java
      // add graphics overlays to the scene view
      sceneView.getGraphicsOverlays().addAll(Arrays.asList(
        drapedBillboardedOverlay, relativeOverlay, absoluteOverlay, relativeToSceneOverlay));
```